deps/linenoise/README.markdown

```diff
@@ -59,7 +59,7 @@ Linenoise is very easy to use, and reading the example shipped with the
 library should get you up to speed ASAP. Here is a list of API calls
 and how to use them.
 
-    char *linenoise(const char *prompt);
+    char *linenoise(const char *prompt, const char *initial_buf, const int initial_buf_len);
```

Comment:
I think it's nice to keep the very simple API where `linenoise` takes just one argument and gives an empty initial buffer. We can add another function with 3 arguments, `linenoiseInitBuf` or `linenoise2` or whatever.

Reply:
yeah you're right, I'll change it 👍

Reply:
hey @zuiderkwast what do you think of [this](https://github.com/redis/redis/pull/12543/commits/ba2d0f197379128a62c573a01a9798f0ff9f034e#diff-1355a467962b08c03a030ff847b78166cf4aed4ea191463c92a46c9e13387533R1159)?

it keeps the `linenoise()` function the way it was, but also adds a `linenoiseWithBuffer()` function as well.  `linenoise()` just wraps `linenoiseWithBuffer()` and calls it with no buffer

Reply:
exactly